cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/PhaseTapChangerConversion.java

```diff
@@ -350,15 +351,16 @@ private void addSteps(
 
         for (int i = 0; i < alphas.size(); i++) {
             double alpha = alphas.get(i);
+            double alphaRatio = -alpha;
```

Comment:
I am not sure to understand the bug : at this stage of the code, the tables ````rhos```` and ````alphas```` are in the IIDM convention. In fact, you want to go back to CGMES convention to call the functions ````getStepXforAsymmetrical```` and ````getStepXforSymmetrical````. Am I right ?

Reply:
You are right. The functions `getStepXforAsymmetrical` and `getStepXforSymmetrical` are coded using CGMES convention. 
Instead of changing the `getStepXforAsymmetrical` and `getStepXforSymmetrical`  to the IIDM convection i propose to spend a couple of hours to refactorize this code following a schema closer to the new CGMES conversion. It will be a small change.
In this schema i will work with CGMES convention (angles and ratios) until we set this information to IIDM. At this point i will change to IIDM convention and i will move to side 1 if it is necessary.
I know that the current conversion will be changed by the new one but at least after the refactorization will be easier to compare both. 
Are you agree?

Reply:
I agree, it will more comprehensive I think.

Reply:
To conclude: there is a bug (which I understand well) and you propose a quick solution but you propose to refactorize this code in a next PR ?

Reply:
I have done a refactorization of this part of the code to be more clear. We are testing now the refactored code with the conformity cases. After the test i will update this PR. The idea is to include the refactorization in this PR.